test/parallel/test-worker-title-prefix.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const assert = require('assert');
+const { Worker, isMainThread } = require('worker_threads');
+
+const { Session } = require('inspector');
+
+if (isMainThread) {
+  const titlePrefix = 'Hello Thread ';
+  new Worker(__filename, {
+    workerData: 'Test Worker',
+    titlePrefix,
+  });
+
+  const session = new Session();
+  session.connect();
+  session.on('NodeWorker.attachedToWorker', ({ params: { workerInfo } }) => {
+    const id = workerInfo.id;
+    const expectedTitle = `${titlePrefix}Worker ${id}`;
```

Comment:
`Hello ThreadWorker` just feels a bit odd (specifically, not having a space between the prefix and `Worker`). I would almost prefer a model like `Worker {id} [label]` ... e.g. `Worker 1 [Hello Thread]`. Consider this non-blocking tho.

Reply:
Yes i am updating the name formation as per this https://github.com/nodejs/node/pull/46832#issuecomment-1446147774